tests/tornado/test_peer.py

```diff
@@ -324,8 +324,9 @@ def test_choose_with_target_hostport(hostports):
     target = '1.0.0.1:9000'
     for _ in hostports:
         peer = peer_group.choose(hostport=target)
+        assert target in peer_group.hosts
         assert target == peer.hostport
-        assert peer_group.peer_heap.size() == n
+        assert peer_group.peer_heap.size() == n - 1
```

Comment:
`n` is set to `len(hostports) + 1`, but the only use of `n` does `n - 1`, maybe we should do: `n = len(hostports)`?